src/templates/atoms.njk

```diff
@@ -879,14 +879,22 @@
                 <div class="a-input">
                     <label class="a-input__label" for="text-field">Text field</label>
                     <input type="text" name="text-field" id="text-field" placeholder="Placeholder" aria-describedby="text-field-description" />
-                    <small id="text-field-description">Input description</small>
+                    <div class="a-input__info">
+                        <small class="a-input__helper-text">Input description</small>
+                        <div class="a-input__info-spacer"></div>
+                        <small class="a-input__character-counter">0 / 200</small>
```

Comment:
Jeroen had gevraagd om de counter mee op te nemen in de voorbeelden, maar niet als standaard, dus graag een apart voorbeeld hiervoor en liefst ergens meer onderaan.